### PR TITLE
Update README according to new template

### PR DESCRIPTION
**Pull request by hermancollin**
This PR aims to update the README according to the new template (https://github.com/neuropoly/axondeepseg/wiki/Template-for-Readme-of-new-models).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,55 @@
 # model_seg_rat_axon-myelin_bf
+---
+## Model overview
+![bf model preview img](bf_model_preview.png)
```
Comment (mariehbourget) [inline README.md]: We may have to ask permissions to publish this image as the dataset is not public.

File: README.md [hunk 2]
```diff
@@ -1,9 +1,55 @@
...
+git clone https://github.com/axondeepseg/default-BF-model
+```
+
+### Get the data
+- git@data.neuro.polymtl.ca:datasets/data_axondeepseg_bf_training
+- commit f833b905c2cb221d45b2ef5ac2fad1100e70b410
+
+```
+git clone git@data.neuro.polymtl.ca:datasets/data_axondeepseg_bf_training
+cd dataset-used-to-train-the-model
```
Comment (mariehbourget) [inline README.md]:
```suggestion
cd data_axondeepseg_bf_training
```

**Comment by mariehbourget** [comment]
Here is the version file for BF:
[version_info.zip](https://github.com/axondeepseg/default-BF-model/files/8181581/version_info.zip)

This one is a little bit more complicated because the dataset was not annexed at training time, in the README, I put the link on duke (the version file only have the local link I used in rosenberg).

File: README.md (hunk 1 above)
Comment (hermancollin) [reply README.md]: Opened an issue for this https://github.com/axondeepseg/default-BF-model/issues/3

File: README.md [hunk 3]
```diff
@@ -1,9 +1,47 @@
...
+
+### Get the data
+The dataset used to train this model is hosted on git-annex at `data.neuro.polymtl.ca:datasets/data_axondeepseg_bf_training`. The specific dataset revision hash used for training is documented in the *version_info.log* file.
+
+### Train this model
+To train the model, please first update the following fields in the aforementioned JSON configuration file:
+- `gpu_ids`: specific to your hardware
+- `path_output`: where the model will be saved
+- `loader_parameters:path_data`: path to training data
+- `loader_parameters:bids_config`: path to the custom bids config located in `ivadomed/config/config_bids.json`
```
Comment (jcohenadad) [inline README.md:39]: why is this needed if a custom JSON is provided with the repository?

File: README.md [hunk 4]
```diff
@@ -1,9 +1,47 @@
...
+This model was trained and tested with [ivadomed](https://ivadomed.org). We recommend you install ivadomed in a virtual environment to reproduce the original training steps. The specific revision hash of the version used for training is documented in the *version_info.log* file.
+
+### Clone this repository
+You will need the *model_seg_rat_axon-myelin_bf.json* configuration file located in this repo.
+```
+git clone https://github.com/axondeepseg/default-BF-model
+```
+
+### Get the data
+The dataset used to train this model is hosted on git-annex at `data.neuro.polymtl.ca:datasets/data_axondeepseg_bf_training`. The specific dataset revision hash used for training is documented in the *version_info.log* file.
```
Comment (mariehbourget) [inline README.md]: The dataset was not yet annexed when I did the training of this model so the version_info.log file just says "Dataset is not Annexed."
The dataset version I used is the same as the current one (now annexed), wo we could add the commit of the current version here.

Comment (hermancollin) [reply README.md]: @mariehbourget do you mean adding the commit in the README or in the version file?

Comment (mariehbourget) [reply README.md]: I would keep the version file intact and add the git-annex commit in the README for this specific model.

**Review by mariehbourget — APPROVED** [review]
Looks good!
Thanks @hermancollin!
